src/pysource/sensitivity.py

```diff
@@ -62,6 +67,9 @@ def crosscorr_time(u, v, model, **kwargs):
     model: Model
         Model structure
     """
+    if model.is_viscoacoustic:
+        return u[0].dt2 * v[0]
```

Comment:
so looks like the imaging condition is the same as acoustic so it probably supports everything (isic, dft) out of the box, i.e can probably just keep all of this as is. You probably need to just rewrite the pde with `1/(m*rho)` rather than `1/m`

Reply:
Allows you to also skip all the extra model arg in `wave_utils`

Reply:
I rewrote the PDE considering  `1/(m*rho)` and did the tests and obtained the following results: 

Testing `isic=false`

Acoustic:
**Basic source for linearized modeling** 
```
basic_src return:
-dm(x, y)*irho(x, y)*Derivative(u(t, x, y), (t, 2))
```

**Cross correlation of forward and adjoint wavefield**
```
crosscorr_time return: 
dt*irho(x, y)*u(time, x, y)*Derivative(v(t, x, y), (t, 2))
```

Viscoacoustic:
**Basic source for linearized modeling**
```
basic_src return:
-dm(x, y)*irho(x, y)*Derivative(u(t, x, y), (t, 2))
```

**Cross correlation of forward and adjoint wavefield**
```
crosscorr_time return:  
v(t, x, y)*Derivative(u(time, x, y), (time, 2))
```

Notice that the crosscorr_time expression of viscoacoustic is different from acoustic, but the test pass when isic=false.  

Testing `isic=true`

For the acoustic and viscoacoustic are equal the expressions. 
However, the test fails for viscoacoustic case:

**ISIC source for linearized modeling**
```
isic_src return: 
-dm(x, y)*irho(x, y)*m(x, y)*Derivative(u(t, x, y), (t, 2)) + Derivative(dm(x, y)*
irho(x,y)*Derivative(u(t, x, y), x), x) + Derivative(dm(x, y)*irho(x, y)*
Derivative(u(t, x, y), y), y)
```
**Inverse scattering imaging condition**
```
isic_time return: 
dt*(m(x, y)*u(time, x, y)*Derivative(v(t, x, y), (t, 2)) + Derivative(u(time, x, y), x)*
Derivative(v(t, x, y), x) + Derivative(u(time, x, y), y)*Derivative(v(t, x, y), y))*
irho(x, y)
```
As I don’t have expertise about the` isic` theory that you implemented, I can’t say if the expression should be the same.  

Could you help me with this?

Reply:
First of all, thanks for your reply.
> It's not, you can move the derivative from `u` to `v` it's equivalent. And it has the advantage to be much better for some options such as time subsampling where having the derivative on `u` would be very inaccurate
yes, I know that. But I mean concerning to `dt*irho(x, y)`. Although there isn't this term in `crosscorr_time` of viscoacoustic the test pass (which is weird because should pass if the `basic_src` and `crosscorr_time` were equivalent to both (acoustic and viscoacoustic) ). 

However, for the second case (when `isic=true`)  the` isic_src` and `isic_time` of viscoacoustic are equivalent to acoustic, but the test fails.

Reply:
In what way does the test fail.


Thanks a lot for your patience a bearing with all my comments and discussions.

Reply:
yes, I will do that.

Reply:
For `isic` case I think need to remove `r` from the wavefield for the linearized source and should work

Reply:
I don't think that's the reason because we had already removed `r` from the wavefield.